torch/_refs/linalg/__init__.py

```diff
@@ -56,6 +58,17 @@ def check_norm_dtype(dtype: Optional[torch.dtype], x_dtype: torch.dtype, fn_name
 from torch._decomp import register_decomposition
 
 
+# TODO: Defined as an alias in the OpInfo, but has different default arguments.
+def diagonal(
```

Comment:
Could you either fix this or open an issue?

Reply:
https://github.com/pytorch/pytorch/issues/85419